pkg/cmd/experimental/registry/registry.go

```diff
@@ -140,6 +143,18 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 		label = valid
 	}
 
+	nodeSelector := map[string]string{}
+	if len(cfg.Selector) > 0 {
+		valid, remove, err := app.LabelsFromSpec(strings.Split(cfg.Selector, ","))
+		if err != nil {
+			glog.Fatal(err)
```

Comment:
Yeah, the router code doesn't bubble up the errors - that needs to be fixed sometime. 

As @liggitt  mentioned, the preferred way would be to bubble up the errors and have the top level code (```cobra.Command :: Run``` handle it)  ala: 
```  
 Run: func(cmd *cobra.Command, args []string) {  
        err := processCommand(f, options, cmd, args, out)  
        cmdutil.CheckErr(err)  
},  
```